### DISCUSSION TITLE
Email notifications for low stock items

### DISCUSSION THREAD

**Discussion by palagap** [inventree/InvenTree#3066] [Q&A] [answered]
Hi there,

I'm curious if there's a way to enable email notifications for parts when they are low in stock. I see that it's possible to turn on notifications for a part:

<img width="334" alt="Screen Shot 2022-05-24 at 9 48 01 AM" src="https://user-images.githubusercontent.com/40808444/170089679-a852d03d-11b9-46d4-b821-e74fee568203.png">

This then results in a notification in the top left of the screen:

<img width="400" alt="Screen Shot 2022-05-24 at 9 48 12 AM" src="https://user-images.githubusercontent.com/40808444/170089749-3ca9304b-88f0-4223-9f90-da2b4bbcaae8.png">

 Is there a way, though, to automatically configure that an email notification will result if any part is low in stock (without manually turning on notifications for each part)?

Thanks.

**Comment by matmair** [comment, marked as answer]
There is currently no way to subscribe to all parts. Notification delivery via mail is subject to the general subscription setting.

**Reply by palagap** [reply]
Got it - where can I access the general subscription setting? I see that in settings I can add an email address, but it doesn't look like that enables notifications. I see a notifications tab in settings, but it doesn't appear to have any configurations available.

**Reply by matmair** [reply]
@palagap have you enabled mail notifications in the global settings?

**Reply by palagap** [reply]
I've clicked through all the panels in the image below, but I don't see a mail notifications option (this is the 0.7.0 dev version)

<img width="238" alt="Screen Shot 2022-05-25 at 12 40 12 PM" src="https://user-images.githubusercontent.com/40808444/170353970-005324c5-a083-43f8-9988-f6d1ea9def33.png">

The notification panel looks like:

<img width="1915" alt="Screen Shot 2022-05-25 at 12 41 38 PM" src="https://user-images.githubusercontent.com/40808444/170354159-3ef4b976-d3cb-43a4-8a45-d0ddedcc3dc3.png">

**Reply by matmair** [reply]
@palagap it seems like you do not have enabled the CoreNotificationPlugin - could you check if that is disabled?

**Reply by palagap** [reply]
That could be it - where can I access the CoreNotificationPlugin? I'm using the demo version - not sure if that's an issue.

**Reply by matmair** [reply]
@palagap plugins and therefore emails are turned off on the demo version - would be pretty bad if anyone could send anybody mails via our demo.

**Reply by palagap** [reply]
I see, that makes sense. I'll take a look at the non-demo version I'm using to see if I can get the notifications to work. Thanks!
